src/ctelnet.cpp

```diff
@@ -1161,26 +1161,26 @@ void cTelnet::setGMCPVariables(const QString& msg)
     QString var;
     QString arg;
     if (msg.indexOf('\n') > -1) {
-        var = msg.section("\n", 0, 0);
-        arg = msg.section("\n", 1);
+        var = msg.section(QStringLiteral("\n"), 0, 0);
```

Comment:
I think anywhere where we are looking at line feeds in `QString`s we can use `QChar::LineFeed` which nicely bypasses any raw string literal concerns of `QT_NO_CAST_{FROM|TO}_ASCII`...

Reply:
Fixed